javascript/code-challenges/challenges-06.test.js

```diff
@@ -94,7 +90,9 @@ console.log(people[1].isAuthor) prints true
 ------------------------------------------------------------------------------------------------ */
 
 const setStatusAsAuthor = (people) => {
-  // Solution code here...
+  people.forEach((person) => {
+    person.isAuthor = true;
+  });
 };
```

Comment:
This works. The only reason it's not giving a passing test is because the test for this problem was still x'd out at the bottom of the file. I'll go ahead and give you extra credit for solving it.